src/EditorFeatures/Test2/FindReferences/FindReferencesTests.AccessorSymbols.vb

```diff
@@ -0,0 +1,507 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
+    Partial Public Class FindReferencesTests
```

Comment:
>FindReferencesTests [](start = 25, length = 19)

If not tested already, may be good to try with object initializers: `new C() { P = ... }` or maybe anonymous objects. Also in attributes: `SomeAttribute(P = ...)`.
Also, as Joseph mentioned, crefs in xml docs would be a good scenario.

May be worth testing on accessors for C# indexers.
I think in VB indexers can be named, right? If so, consider testing named indexers in VB (from declaration and usage)?

Reply:
good test ideas.  will add them asap!

Reply:
Ok.  tests added.  Note: i also cleaned things up a lot here between how these two reference finders work.  it looks like a large change, but essentially the only difference is that reference-finders now return the Location objects of matches (as before) *and* the nodes the result was for.  This greatly simplified a lot of the code in this area as we were having to reverse engineer that from the other finder when calling into the first.